### DISCUSSION TITLE
How to detect whether the layer is visible

### DISCUSSION THREAD

**Discussion by zxc155226522** [ImageMagick/ImageMagick#7073] [Help]
### ImageMagick version

ImageMagick-7.1.1-6-portable-Q8-x64 

### Operating system, version and so on

How to detect whether the layer is visible

### Description

How to detect whether the layer is visible

**Comment by urban-warrior** [comment]
Add `-debug` to your command line to determine if a layer is invisible.  Or try this command:

```
$ identify -verbose myImage.pdf
```

If the Compose property is None, the layer is invisible.
